src/common/context/forms/index.tsx

```diff
@@ -28,25 +30,32 @@ export const useFormsContext = (): FormsContext => useContext<FormsContext>(Form
 export const FormsContextProvider: FunctionComponent = ({ children }) => {
   const [activeFormIndex, setActiveFormIndex] = useState<number | undefined>(undefined);
   const [forms, setForms] = useState<FormEntry[]>([]);
+  const { config } = useConfigContext();
+
+  const currentForm = activeFormIndex === undefined ? undefined : forms[activeFormIndex];
+  const currentFormData = currentForm ? currentForm.data : undefined;
+  const currentFormOwnership = currentForm ? currentForm.ownership : undefined;
+  const currentFormTemplate = currentForm ? config?.forms[currentForm?.templateIndex] : undefined;
 
   const newForm = (templateIndex: number): void => {
     const newIndex = forms.length;
+    const newFormTemplate = config?.forms[templateIndex];
+
     setForms([
       ...forms,
       {
         templateIndex,
-        data: {} as any, // eslint-disable-line @typescript-eslint/no-explicit-any
+        data: {
+          formData: newFormTemplate?.defaults || {},
+          schema: newFormTemplate?.schema,
+        } as any, // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
any is wrong here right ?

it's an object with properties `formData` and `schema` that are any

Reply:
yep right, removed